day03/kim/index.js

```diff
@@ -0,0 +1,12 @@
+// 呼叫 Dom
+const input = document.querySelectorAll('.controls input')
+
+// 監聽事件：改變(聽三組)
+input.forEach(element => 
```

Comment:
try addEventListener on `.controls`

Reply:
Morning, Thank you for the advice. 😊
But, I not sure the reply mean.

It's mean, 
> Try to control different DOM,  to complete this question ?

I guess the opening will be : 

```
const control = document.querySelectorAll('.controls')
control.addEventListener ...

control Spacing
control Blur
control Base Color
```
And I try to `console` `control`
![D249C20E-4AEF-480A-BB78-80AB7A281D51_4_5005_c](https://user-images.githubusercontent.com/87320011/148709265-687554c7-dc83-402a-bedf-841be824ea58.jpeg)

I guess ,maybe I will need to search `children` `classList` to complete,
but, I not sure the target I should do.

Am I toward the right direction ?
If you have free time, could you give me more hint about the reply. Thank you ^^

Reply:
Yes, you can add event listener on the `ancestor` instead of add listener on individual children.

check [Event bubbling](1) for more detail.

[1]: (https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Building_blocks/Events#event_bubbling_and_capture).

Reply:
https://ithelp.ithome.com.tw/articles/10219793
記得叫做event proxy，但是一時找不到文章介紹，可能關鍵字有錯
再有大量eventlistener時候用的技巧